plugins/search/providers/elastic_search/lib/model/filters/ESearchQueryFromFilter.php

```diff
@@ -54,7 +54,7 @@ public function retrieveElasticQueryEntryIds(baseObjectFilter $filter, kPager $p
 
 		$entrySearch = new kEntrySearch();
 		$entrySearch->setFilterOnlyContext();
-		$elasticResults = $entrySearch->doSearch($query, array(), null, $pager, null);
+		$elasticResults = $entrySearch->doSearch($query, ESearchCaptionQueryFromFilter::$validStatuses, null, $pager, null);

```

Comment:
The word caption cannot be mentioned in this file. 
Find a way to move it to derived class.